src/ServicePulse.Host/vue/src/components/failedmessages/LastTenOperations.vue

```diff
@@ -1,45 +1,118 @@
+<script setup>
+import { ref, onMounted } from "vue";
+import NoData from "../NoData.vue";
+import { useFetchFromServiceControl } from "../../composables/serviceServiceControlUrls";
+import TimeSince from "../TimeSince.vue";
+
+const historicOperations = ref([]);
+const showHistoricRetries = ref(false);
+
+function getHistoricOperations() {
```

Comment:
It will be good to move the calls out to the js file like the other pages

Reply:
As we discussed, we are going to leave this for now.  We can pull this api out into a separate file later if we need to.